### PR TITLE
add categories seed

### PR DESCRIPTION
**Pull request by SuddenGunter**

### CODE REVIEW

File: storage/seed/seed.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+       (1, 'Unknown', 'creditcard', current_timestamp(0)),
+       (127, 'Ignored', 'clear', current_timestamp(0)),
+       (2, 'Health', 'cross.fill', current_timestamp(0)),
+       (3, 'Grocery', 'leaf', current_timestamp(0)),
+       (4, 'Tax', 'crown', current_timestamp(0)),
+       (5, 'Rent', 'house', current_timestamp(0)),
+       (6, 'Internet', 'network', current_timestamp(0)),
+       (7, 'Clothes', 'eyeglasses', current_timestamp(0)),
+       (8, 'IKEA', 'Wrench', current_timestamp(0)),
+       (9, 'Caffee', 'list.dash', current_timestamp(0)),
```
Comment (flerka) [inline storage/seed/seed.sql:12]: Is list.dash correct logo name?

File: category/category.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package category
+
```
Comment (flerka) [inline category/category.go:2]: Are you sure that category identifier should be hard coded in database and in code?

File: storage/seed/seed.sql (hunk 1 above)
Comment (SuddenGunter) [reply storage/seed/seed.sql:12]: yeap

File: category/category.go (hunk 2 above)
Comment (SuddenGunter) [reply category/category.go:2]: I'd prefer to seed table from the code, but do not want to spend time now. 
I've created separate issue for it https://github.com/lungria/spendshelf-backend/issues/43

**Pull request merged**